src/Billing/Controllers/FreshsalesController.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using Bit.Core.Enums;
+using Bit.Core.Repositories;
+using Bit.Core.Settings;
+using Bit.Core.Utilities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Bit.Billing.Controllers
+{
+    [Route("freshsales")]
+    public class FreshsalesController : Controller
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly IOrganizationRepository _organizationRepository;
+        private readonly ILogger _logger;
+        private readonly GlobalSettings _globalSettings;
+
+        private readonly string _freshsalesApiKey;
+
+        private readonly HttpClient _httpClient;
+
+        public FreshsalesController(IUserRepository userRepository,
+            IOrganizationRepository organizationRepository,
+            IOptions<BillingSettings> billingSettings,
+            ILogger<FreshsalesController> logger,
+            GlobalSettings globalSettings)
+        {
+            _userRepository = userRepository;
+            _organizationRepository = organizationRepository;
+            _logger = logger;
+            _globalSettings = globalSettings;
+
+            _httpClient = new HttpClient
+            {
+                BaseAddress = new Uri("https://bitwarden.freshsales.io/api/")
+            };
+
+            _freshsalesApiKey = billingSettings.Value.FreshsalesApiKey;
+
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue(
+                "Token",
+                $"token={_freshsalesApiKey}");
+        }
+
+
+        [HttpPost("webhook")]
+        public async Task<IActionResult> PostWebhook([FromHeader(Name = "Authorization")] string freshsalesApiKey,
```

Comment:
let's try to match the FD implementation and name the key parameter as `key` vs. `freshsalesApiKey` which could be confused with the configured value.